### PR TITLE
0620 change community to opensource also fix relup

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

**Review by ieQu1 — APPROVED** [review]

File: scripts/update_appup.escript [hunk 1]
```diff
@@ -377,7 +370,8 @@ ensure_version(Version, OldInstructions) ->
 
 contains_version(Needle, Haystack) when is_list(Needle) ->
     lists:any(
-      fun(Regex) when is_binary(Regex) ->
+      fun(<<"*">>) -> true; %% TODO: delete after we pass esockd 5.8.4
```
Comment (thalesmg) [inline scripts/update_appup.escript:373]: Maybe this can be removed?  `esockd` seems to be at 5.9.3.

**Review by thalesmg — APPROVED** [review]

**Comment by zmstone** [comment]
Will fix relup CI in another PR.

File: apps/emqx_gateway/src/emqx_gateway.appup.src [hunk 2]
```diff
@@ -0,0 +1,9 @@
+%% -*- mode: erlang -*-
+%% Unless you know what you are doing, DO NOT edit manually!!
+{VSN,
+  [{"0.1.0",[{load_module,emqx_exproto_pb,brutal_purge,soft_purge,[]}]},
+   {"5.0.0",[{load_module,emqx_exproto_pb,brutal_purge,soft_purge,[]}]},
```
Comment (thalesmg) [inline apps/emqx_gateway/src/emqx_gateway.appup.src:5]: This seems far from the current `0.1.0`?

**Review by thalesmg — DISMISSED** [review]

**Review by ieQu1 — APPROVED** [review]

**Comment by zmstone** [comment]
Will fix ci in follow up pr

**Pull request merged**